packages/styles/scss/patterns/sub-patterns/content-block/_content-block.scss

```diff
@@ -11,72 +11,71 @@
 @import '../../blocks/feature-card/feature-card';
 
 @mixin content-block {
+  .#{$prefix}--content-block-inverse {
+    background-color: $inverse-02;
+    color: #ffffff;
+    padding-top: $carbon--layout-03;
+    padding-bottom: $carbon--layout-05;
+    @include carbon--breakpoint('md') {
+      padding-top: $carbon--layout-05;
+      padding-bottom: $carbon--layout-06;
+    }
+    @include carbon--breakpoint('max') {
+      padding-bottom: $carbon--layout-07;
+    }
```

Comment:
@andysherman2121 doesn't seem like any of these are needed for the modifier?

Reply:
These spacings are all applied to the normal `.#{$prefix}--content-block` class so when you switch to the inverse modifier, it gets rid of necessary spacing

Reply:
adjusted